### PR TITLE
Docker: Upgrade to Ubuntu 20.04 in Dockerfiles

### PR DESCRIPTION
**Pull request by aknuds1**
**What this PR does / why we need it**:
Upgrade to the new Ubuntu LTS, 20.04, in Dockerfiles. Also upgrade build pipeline tool, in order to get its support for new Ubuntu.

### CODE REVIEW

**Review by sakjur — APPROVED** [review]
🎉 Nice!

**Comment by sakjur** [comment]
https://github.com/grafana/grafana/blob/master/packaging/docker/build.sh#L67 as well

**Comment by aknuds1** [comment]
> https://github.com/grafana/grafana/blob/master/packaging/docker/build.sh#L67 as well?

Thanks @sakjur for catching that! Fixed.

**Review by papagian — APPROVED** [review]
LGTM

**Review by bergquist — APPROVED** [review]
:+1:

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 ## Breaking changes
 
 - **Removed PhantomJS**: PhantomJS was deprecated in [Grafana v6.4](https://grafana.com/docs/grafana/latest/guides/whats-new-in-v6-4/#phantomjs-deprecation) and starting from Grafana v7.0.0, all PhantomJS support has been removed. This means that Grafana no longer ships with a built-in image renderer, and we advise you to install the [Grafana Image Renderer plugin](https://grafana.com/grafana/plugins/grafana-image-renderer).
+- **Upgraded Ubuntu to 20.04**: Our Ubuntu based images have been upgraded to Ubuntu [20.04 LTS](https://releases.ubuntu.com/20.04/).
```
Comment (marefr) [inline CHANGELOG.md:6]:
```suggestion
- **Docker**: Our Ubuntu based images have been upgraded to Ubuntu [20.04 LTS](https://releases.ubuntu.com/20.04/).
```

**Review by marefr — APPROVED** [review]
LGTM

**Pull request merged**